src/biokbase/narrative/tests/test_jobmanager.py

```diff
@@ -277,15 +278,51 @@ def test_list_jobs_html(self):
         jobs_html = self.jm.list_jobs()
         self.assertIsInstance(jobs_html, HTML)
         html = jobs_html.data
-        for job_id in TEST_JOB_IDS:
-            self.assertIn("<td>" + job_id + "</td>", html)
-        self.assertIn("<td>NarrativeTest/test_editor</td>", html)
-        self.assertIn("<td>2019-08-26 ", html)
-        self.assertIn(":54:48</td>", html)
-        self.assertIn("<td>fake_test_user</td>", html)
-        self.assertIn("<td>completed</td>", html)
-        self.assertIn("<td>Not started</td>", html)
-        self.assertIn("<td>Incomplete</td>", html)
+
+        counts = {
+            "status": {},
+            "app_id": {},
+            "batch_id": {},
+            "user": {},
+            # "incomplete": 0,
+        }
+
+        n_not_started = 0
+        n_incomplete = 0
+        for job in TEST_JOBS.values():
+            for param in ["status", "user"]:
+                if param in job:
+                    value = job[param]
+                    if value not in counts[param]:
+                        counts[param][value] = 0
+                    counts[param][value] += 1
+
+            app_id = job["job_input"]["app_id"]
+            if app_id not in counts["app_id"]:
+                counts["app_id"][app_id] = 0
+            counts["app_id"][app_id] += 1
+
+            if "finished" not in job:
+                n_incomplete += 1
+            if "running" not in job:
+                n_not_started += 1
+
+        for job_id in ALL_JOBS:
+            self.assertIn(f"<td>{job_id}</td>", html)
+
+        for param in counts:
+            for value in counts[param]:
+                self.assertIn("<td>" + str(value) + "</td>", html)
+                value_count = html.count("<td>" + str(value) + "</td>")
+
+                self.assertEqual(counts[param][value], value_count)
+
+        if n_incomplete:
+            incomplete_count = html.count("<td>Incomplete</td>")
+            self.assertEqual(incomplete_count, n_incomplete)
+        if n_not_started:
+            not_started_count = html.count("<td>Not started</td>")
+            self.assertEqual(not_started_count, n_not_started)
```

Comment:
check that there are as many incomplete jobs and jobs that have not started running as we would expect from the raw data